helm/sematic-server/templates/server-role.yaml

```diff
@@ -14,3 +14,11 @@ rules:
 - apiGroups: [""]
   resources: ["pods"]
   verbs: ["get", "watch", "list"]
+- apiGroups: ["apps"]
+  resources: ["deployments"]
+  verbs: ["get", "watch", "list"]
+{{- if .Values.rbac.manage_ray }}
```

Comment:
Maybe there is a better boolean to use here. I need to make it so that the Ray plugins are configurable via helm chart, that's a TODO.